example/settings.py

```diff
@@ -82,9 +82,15 @@
 
 ES_DISABLED = False
 
-ES_URLS = ['http://localhost:9200']
+ES_URLS = ['http://elasticsearch.local:9200']
```

Comment:
Not 100% sure if this is what we're looking for here, but you can set environment variables in `docker-compose` and then access that in `settings.py`.
- [Setting environment variables in `docker-compose`](https://github.com/theonion/starwipe-cms/blob/master/resources/docker/docker-compose.yml#L9)
- [Accessing environment variables in a settings file](https://github.com/theonion/starwipe/blob/ea59243679ea9c9dec7186c9654d92ad666d3612/starwipe/settings/testing.py#L6)